robottelo/ui/navigator.py

```diff
@@ -20,6 +20,7 @@ def menu_click(self, top_menu_locator, sub_menu_locator,
                 "arguments[0].click();",
                 tertiary_element,
             )
+        self.browser.refresh()
```

Comment:
Tried few more alternative solutions and no luck. Proposed approach fixes all failures that connected to different aspects of page reload procedure of 6.2 application. Any other ideas doesn't help.